features/post/post-comment/post-comment.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactNode } from 'react';
+
+import useRelativeTime from '@/shared/hooks/useRelativeTime';
+import { useTranslation } from '@/shared/hooks/useTranslation';
+import { Avatar, Typography } from '@/shared/ui';
+import clsx from 'clsx';
+
+import s from './post-comment.module.scss';
+
+export type PostCommentT = {
+  avatar?: string;
+  children?: ReactNode;
+  childrenInfoSection?: ReactNode;
+  createdAt: string;
+  iconElement?: ReactNode;
+  likesCount: number;
+  text: string;
+  textWidth?: string;
+  userName: string;
+};
+
+export const PostComment = (props: PostCommentT) => {
+  const {
+    avatar,
+    children,
+    childrenInfoSection,
+    createdAt,
+    iconElement,
+    likesCount,
+    text,
+    textWidth,
+    userName,
+  } = props;
+  const relativeTime = useRelativeTime(createdAt);
+  const { t } = useTranslation();
+
+  const getInitials = (userName: string): string => {
+    const words = userName.split(' ');
+    const initials = words.map(word => word.charAt(0).toUpperCase());
+
```

Comment:
The `Avatar` and username should be wrapped in a `Next/Link` component, so that it is possible to easily navigate to the profile of the interested user.